com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -386,6 +386,7 @@ internal void InternalNetworkStart()
         internal readonly List<INetworkVariable> NetworkVariableFields = new List<INetworkVariable>();
 
         private static HashSet<NetworkObject> s_Touched = new HashSet<NetworkObject>();
+        private static HashSet<NetworkObject> s_TouchedThisClient = new HashSet<NetworkObject>();
```

Comment:
if you look at my transform test PR https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/868, I've moved these static vars to its own component, since this was making tests flaky.